### ISSUE TITLE
Address missing <legend> tags

### ISSUE THREAD

**Issue by wesley-dean-gsa** [GSA/usagov-benefits-eligibility#357]
# User Story

As a BEARS **end-user who uses assistive technologies**, in order to **more easily navigate forms on the site**, I would like **`<legend>` tags added to form fieldsets**.

Pre-conditions:

- [x] SonarQube has identified 3 form fieldsets where there are no `<legend>` tags
   - [x] [components/CheckBox.vue](https://github.com/GSA/usagov-benefits-eligibility/blob/main/components/CheckBox.vue#L2-L17)
   - [x] [components/DropDown.vue](https://github.com/GSA/usagov-benefits-eligibility/blob/main/components/DropDown.vue#L2-L32)
   - [x] [pages/_lifeEvent.vue](https://github.com/GSA/usagov-benefits-eligibility/blob/main/pages/_lifeEvent.vue#L140-L163)
   - [x] [components/summaryBox.vue](https://github.com/GSA/usagov-benefits-eligibility/blob/main/components/SummaryBox.vue#L9-L33)

Acceptance Criteria:

- [x] `<legend>` tags have been added to the forms

Definition of Done:

- [x] Code complete
- [x] Tests coverage is greater than team benchmark (90% goal)
- [x] Security scans passed
- [x] Acceptance Criteria is met and it works as expected
- [x] Accessibility tested
- [x] Build process and deployment is automated and repeatable
- [x] Load testing/performance testing
- [x] Self Documentation whenever possible
- [x] Feature toggles if appropriate
- [x] Deployed to staging
- [ ] Usability testing
- [ ] PR approved / Peer reviewed
- [ ] PO approved

**Comment by wesley-dean-gsa** [comment]
see also #337

**Comment by wesley-dean-gsa** [comment]
https://github.com/GSA/usagov-benefits-eligibility/blob/main/components/SummaryBox.vue#L9-L33

@fpigeonjr  this morning (Monday, April 4th), SonarQube pointed out a new missing `<legend>` tag so I added it to the list.

**Comment by fpigeonjr** [comment]
perfect, thank you @wesley-dean-gsa.
